feg/gateway/services/session_proxy/servicers/session_controller.go

```diff
@@ -340,20 +341,20 @@ func (srv *CentralSessionController) Enable(
 	ctx context.Context,
 	void *orcprotos.Void,
 ) (*orcprotos.Void, error) {
-	multiError := merrors.NewMulti()
+	multiError := &multierror.Error{}
 	if !srv.cfg.DisableGx {
 		err := srv.policyClient.EnableConnections()
 		if err != nil {
-			multiError.Add(fmt.Errorf("An error occurred while enabling connections; policyClient err: %s", err))
+			multierror.Append(multiError, errors.Errorf("An error occurred while enabling connections; policyClient err: %s", err))
 		}
 	}
 	if !srv.cfg.DisableGy {
 		err := srv.creditClient.EnableConnections()
 		if err != nil {
-			multiError.Add(fmt.Errorf("An error occurred while enabling connections; creditClient err: %s", err))
+			multierror.Append(multiError, errors.Errorf("An error occurred while enabling connections; creditClient err: %s", err))
```

Comment:
at other locations you wrote the following:
```suggestion
			multiError = multierror.Append(multiError, errors.Errorf("An error occurred while enabling connections; creditClient err: %s", err))
```

this feels like unnecessary error prone syntax.

Reply:
Fixed.